Codigo/Protocol_Writes.bas

```diff
@@ -1159,21 +1153,27 @@ End Sub
 ' @param    Message Text to be displayed in the message box.
 ' @remarks  The data is not actually sent until the buffer is properly flushed.
 Public Sub WriteShowMessageBox(ByVal UserIndex As Integer, ByVal Message As String)
-        '<EhHeader>
         On Error GoTo WriteShowMessageBox_Err
-        '</EhHeader>
 100     Call Writer.WriteInt16(ServerPacketID.ShowMessageBox)
 102     Call Writer.WriteString8(Message)
 104     Call modSendData.SendData(ToIndex, UserIndex)
-        '<EhFooter>
         Exit Sub
 
 WriteShowMessageBox_Err:
         Call Writer.Clear
         Call TraceError(Err.Number, Err.Description, "Argentum20Server.Protocol_Writes.WriteShowMessageBox", Erl)
-        '</EhFooter>
 End Sub
 
+Public Function PrepareShowMessageBox(ByVal Message As String)
```

Comment:
Should be a `Sub` as it does not return any value

Reply:
its weird but we usually call this kind of functions as a parameter so every "Prepare...." are functions and are handled as they return the buffer